eslint.config.mjs

```diff
@@ -15,7 +14,6 @@ export default [
         languageOptions: {
             ecmaVersion: 2022,
             sourceType: 'module',
-            parser: babelParser,
             parserOptions: {
                 requireConfigFile: false
             },
```

Comment:
If we don't need the Babel parser any more, can we also drop `parserOptions`? I think we only added this for Babel.